server/api/batch_inventory.py

```diff
@@ -442,19 +443,81 @@ def process_batch_inventory_tabulator_status_file(
     jurisdiction = Jurisdiction.query.get(jurisdiction_id)
     batch_inventory_data = BatchInventoryData.query.get(jurisdiction_id)
 
+    def get_tabulator_id_to_name_dict_for_excel_file(
+        cvr_xml: ElementTree.ElementTree,
+    ) -> Dict[Optional[str], Optional[str]]:
+        namespaces = {"ss": "urn:schemas-microsoft-com:office:spreadsheet"}
+        worksheet = cvr_xml.find(
+            ".//ss:Worksheet[@ss:Name='Tabulator Status']", namespaces
+        )
+        if worksheet is None:
+            raise UserError(TABULATOR_STATUS_PARSE_ERROR)
+
+        tabulator_status_table = worksheet.find("ss:Table", namespaces)
+        if tabulator_status_table is None:
+            raise UserError(TABULATOR_STATUS_PARSE_ERROR)
+
+        columns = tabulator_status_table.findall("Column")
+        if columns is None:
+            raise UserError(TABULATOR_STATUS_PARSE_ERROR)
```

Comment:
Do we use these? If not, why look for them?

Reply:
I used to validate the number of columns was > 0 but removed that check because reading column header values serves the same purpose. Deleted this leftover in https://github.com/votingworks/arlo/pull/2018/commits/abddab989b1569c29fd6d0404c4905b926c11fea#diff-6c612f15bc7de123abfc44b5aead6ce7e0c3a7765f3a0b08543553d068f0e233L460-L462